server/middleware.json

```diff
@@ -26,23 +26,21 @@
     "helmet#hidePoweredBy": {},
     "helmet#ieNoOpen": {},
     "helmet#noSniff": {},
-    "helmet#noCache": { 
+    "helmet#noCache": {
       "enabled": false
     }
   },
-  "session": {
-  },
-  "auth": {
-  },
-  "parse": {
-  },
+  "session": {},
+  "auth": {},
+  "parse": {},
```

Comment:
I see quite few unrelated changes in this file. Is it result of running `slc loopback:*` on this repository? If it's so, can we fix the formatting in the version in the `master` branch?t